README.md

```diff
@@ -1,4 +1,14 @@
-# smarthome-localhomesdk
+# Local Home SDK Virtual Environment
+
+A mock Local Home Platform implemented as stubs for the Local Home SDK
+
+## Usage
+
+1. Place bundled Node Local Fulfillment app into `home-app` folder.
```

Comment:
Can we use a command line argument to point to the bundled code rather than relying on copying it?

Reply:
Will add this